1.preprocessing-data/illumcorrect-data.py

```diff
@@ -26,36 +26,41 @@
 
 
 # ## Perform Illumination Correction on CFReT pilot images
-# 
-# Only one channel (`d0`) was processed in this notebook as an example. All channels are processed using the `illumcorrect-data.ipynb` file
 
 # In[2]:
 
 
 # Set location of input and output locations
-channel_path = pathlib.Path('../0.download-data/Images/')
+data_path = pathlib.Path('../0.download-data/Images/')
 
 save_path = pathlib.Path("IC_Corrected_Images")
 
 # Set the file_extension for the images (if applicable)
 file_extension = '.TIF'
 
+# Plates to process
+plates = ["localhost220512140003_KK22-05-198", "localhost220513100001_KK22-05-198_FactinAdjusted"]
+
 # Channels to process
 channels = ["d0", "d1", "d2", "d3", "d4"]
 
 # Perform illumination correction on each channel seperately using a `for` loop:
-for channel in channels:
-    # print(channel)
-    print("Correcting", channel, "channel images")
+for plate in plates:
+    print("Correcting images from plate", plate)
+    for channel in channels:
+        # print(channel)
+        print("Correcting", channel, "channel images")
 
     # If you want to output the flatfield and darkfield calculations, then put "output_calc=True".
     # If you would like to overwrite any existing images when running this function, set "overwrite=True"
 
-    ICutils.run_illum_correct(
-        channel_path,
-        save_path,
-        channel=channel,
-        output_calc=False,
-        file_extension = '.TIF'
-    )
+        ICutils.run_illum_correct(
+            data_path,
```

Comment:
I try to avoid mixing position and name based arguments. Maybe add data_path and save_path as named arguments for consistency